coffee-shop/src/mocks/cofee.json

```diff
@@ -10,23 +10,23 @@
               "description": "A strong and bold coffee made by forcing steam through ground coffee beans.",
               "price": 2.50,
               "sizes": ["Small", "Medium", "Large"],
-              "image": "/images/espresso.jpg"
+              "image": "Coffee1.png"
```

Comment:
if you will not add images in the path, how should you render images without the path